### ISSUE TITLE
[PM-1173] cleanup jobs failing against Titan gridftp server due to RFC 2818 compliance

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#1287]
Cleanup jobs are failing while connecting to the titan gridftp server. 

Unable to connect to gridftp.ccs.ornl.gov:2811
Authentication failed [Caused by: Operation unauthorized (Mechanism level: [JGLOBUS-56] Authorization failed. Expected &quot;/CN=host/dtn18.ccs.ornl.gov&quot ; target but received &quot;/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=dtn.ccs.ornl.gov&quot;)]
Operation unauthorized (Mechanism level: [JGLOBUS-56] Authorization failed.
Expected &quot;/CN=host/dtn18.ccs.ornl.gov&quot; target but received &quot;/DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=dtn.ccs.ornl.gov&quot;)

**Reporter:** @scallag
**Resolution:** Fixed
**Watchers:**
@rynge
@vahi

**Comment by mayani** [comment]
**Author:** @vahi

we looked at it, and Mats is right. 

This is being triggered because we think JGlobus ( what pegasus-gridftp uses) is not compliant with 

https://docs.globus.org/security-bulletins/2015-12-strict-mode/ 

It does appear that Titan has reissued the host certificates correctly in compliance with the above bulletin. 
However, JGlobus has not been updated. There is an open bug in JGlobus since last year that has not been resolved 
https://github.com/jglobus/JGlobus/issues/159 

Looks like JGlobus is no longer maintained with last release in 2014. 

We decided to revert back to using g-u-c for removing files by transferring a zero byte file for 4.7.5. pegasus-transfer will fall back on g-u-c if pegasus-gridftp fails. 

For 4.8.0 we will remove pegasus-gridftp all together and explore some other client like GFAL2 that pegasus-transfer already uses for xrootd and SRM
